core/src/test/java/io/aiven/klaw/service/UsersTeamsControllerServiceTest.java

```diff
@@ -396,7 +580,102 @@ void showUsers() {}
   void getMyProfileInfo() {}
 
   @Test
-  void addTwoDefaultTeams() {}
+  void addTwoDefaultTeamsSuccess() throws KlawException {
+    int tenantId = 101;
+    String userName = "testuser";
+    String existingTeamName = "existingTeamName";
+    String contactPerson = "contactPerson";
+    ArgumentCaptor<Team> teamCaptor = ArgumentCaptor.forClass(Team.class);
```

Comment:
The Argument Captor can be defined once at a class level and re-used throughout the tests as well.